full-client/test/basic.test.ts

```diff
@@ -1,4 +1,6 @@
 import { Helpers } from '@secret-agent/testing';
+import ResourceType from '@secret-agent/core-interfaces/ResourceType';
```

Comment:
I thought our linter used to catch these?

Reply:
Oh, it is... ha. Should we fix the 3 lint issues in PR?

Reply:
It did... warning not error. Will fix.